src/App.tsx

```diff
@@ -12,7 +17,14 @@ const App = (props: AppProps) => {
         <main className="app veilarbdetaljerfs">
             <StoreProvider fnr={props.fnr}>
                 <Nokkelinfo />
-                <div className="main_grid"></div>
+                <div className="main_grid">
```

Comment:
Fjern innholdet her nå så får vi ned endringene i denne PR'en. Så lager vi en ny branch med kun dette for å ta en siste sjekk før pilotsettingen :)